src/abstractitensornetwork.jl

```diff
@@ -646,6 +654,23 @@ function flatten_networks(
   return flatten_networks(flatten_networks(tn1, tn2; kwargs...), tn3, tn_tail...; kwargs...)
 end
 
+#Ideally this will dispatch to inner_network but this is a temporary fast version for now
+function unflattened_norm_network(tn::AbstractITensorNetwork)
+  tnbra = rename_vertices(v -> (v, 1), data_graph(tn))
+  tndag = copy(tn)
+  for v in vertices(tndag)
+    ITensorNetworks.setindex_preserve_graph!(tndag, dag(tndag[v]), v)
```

Comment:
Shouldn't need `ITensorNetworks.` since we are in that module so `setindex_preserve_graph!` it should be in the namespace.